src/IssueViz/IssueVizPackage.cs

```diff
@@ -45,6 +47,9 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
 
             await IssueVisualizationToolWindowCommand.InitializeAsync(this);
             await NavigationCommands.InitializeAsync(this);
+
+            var componentModel = await GetServiceAsync(typeof(SComponentModel)) as IComponentModel;
+            componentModel.GetService<ISelectedVisualizationValidityMonitor>();
```

Comment:
The monitor will only start when the package was loaded, so until the panel is opened, the service won't run. At the moment this is ok, since the problem is in the panel's state to begin with, but there might be some future problems since technically the SelectionService can contain old issues this way. But since we're only using it for the panel at the moment, it should be ok.